docs/application/native/guides/device/system.md

```diff
@@ -381,6 +381,7 @@ The following table lists the multimedia feature keys:
 | Key                                      | Type   | Description                              |
 |------------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/multimedia.media_codec` | `bool` | The platform returns `true` for this key, if the device supports the media codec feature. |
+| `http://tizen.org/feature/multimedia.stream_recorder` | `bool` | The platform returns `true` for this key, if the device supports the stream recorder feature. |
```

Comment:
Suggestion. I can see these features are in wrong order. How about following the alphabetical order?

Reply:
I will move this feature after multimedia.p** feature as following your suggestion :) thank you

Reply:
This feature is still located before multimedia.p** feature. Would you please check the update?

Also, it would be better to check the location of http://tizen.org/feature/multimedia.transcoder also to arrange this table in alphabetical order.